waku/waku.go

```diff
@@ -1597,6 +1599,10 @@ func (w *Waku) Clean() error {
 	return nil
 }
 
+func (w *Waku) PeerID() peer.ID {
+	return peer.NewPeerRecord().PeerID //FIXME!!!
```

Comment:
@richard-ramos I could not figure out a way how to get the peerId here - any thoughts? Is this even used?

Reply:
WakuV1 is no longer used except for some test units that need to be updated. No need to add this function here. What you could do instead in eth-node/bridge/geth/waku.go:337 is to either return an error, or `panic`.